test/libsolidity/semanticTests/structs/struct_storage_to_memory.sol

```diff
@@ -0,0 +1,25 @@
+pragma experimental ABIEncoderV2;
+
+contract C {
+    struct S {
+        uint32 a;
+        uint128 b;
+        uint256 c;
+    }
+    struct X {
+        uint32 a;
+        S s;
+    }
+
+    X x = X(12, S(42, 23, 34));
```

Comment:
This way, the storage slot of `x` is zero and because of that, the test can accidentally succeed even though we did not really use the storage slot. Please add a "weirdly" sized array above `x` like `uint[37]` to make its storage slot non-trivial.

Reply:
The same argument actually holds for calldata inputs.